config/main.py

```diff
@@ -1014,7 +1048,8 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart):
     if not yes:
         click.confirm(message, abort=True)
 
-    log.log_info("'reload' executing...")
+    global child_pid
+    global PIPE_FILE
```

Comment:
`PIPE_FILE` is a constant. You are not modifying it, so no need to declare it here with `global` keyword.